transform/models/staging/dof_mdsa/stg_footprints_block.sql

```diff
@@ -0,0 +1,13 @@
+with footprints_block as (
+    select 
+f."release",
+f."capture_dates_range",
+f."geometry",
+b."GEOID20",
+b."NAME20"
+from raw_dev.building_footprints.california_building_footprints f
+left join raw_dev.tiger_2022.blocks b
+on st_intersects(f."geometry", b."geometry")
+)
+
+select * from footprints_block
```

Comment:
How computationally expensive is the `block` version of this? If counties took about 90 seconds on a XS, how long did blocks take?

Reply:
Today, Counties took 82 secs, Blocks took 100 secs. Can you help me understand if this information and the difference in speed should change anything I am doing with my code? Blocks is a larger data table (519.7 rows vs 58 rows) so it makes sense to me that the join would take longer and it doesn't seem significantly so despite its size.

Reply:
Ooh, that was understood from your comments above, but this helps paint the picture another way. Thank you Ian!

Reply:
It's 519,723 rows